### ISSUE TITLE
new error

### ISSUE THREAD

**Issue by estimadarocha** [lapicidae/vdr-server#7]
user.err: Mar  3 00:18:11 vdr: [986] [poller.c,118]: epoll_ctl(EPOLL_CTL_DEL) failed: No such file or directory
user.err: Mar  3 00:18:11 vdr: [986] [poller.c,118]: epoll_ctl(EPOLL_CTL_DEL) failed: No such file or directory

**Comment by lapicidae** [comment]
Does the VDR start anyway?
Latest image?

You could also try starting a new test container to narrow down the problem.
e.g.:
`docker run --rm --name vdrtest --env PLUGINS="satip" lapicidae/vdr-server:latest`

**Comment by estimadarocha** [comment]
yes it starts... but i don't think the satip is working ok.... maybe bad compiled or something!

**Comment by michi1972-dev** [comment]
Try host networking. e.g. in docker-compose.yml you can use:
services:
    vdr-server:
        network_mode: "host"

This might help a little bit.

**Comment by greatdane22** [comment]
if it's about sat>ip

you could set portrange in `system/conf.d/50-satip.conf` e.g. 
`--portrange=4010-4029` (see also http://www.vdr-wiki.de/wiki/index.php/Satip-plugin)
*and* 
add this portrange in `docker-compose.yml` under `ports`  i.e. 
`- 4010-4029:4010-4029/udp`
